src/Sylius/Behat/Resources/config/suites.yml

```diff
@@ -41,6 +41,7 @@ imports:
     - suites/ui/inventory/cart_inventory.yml
     - suites/ui/inventory/checkout_inventory.yml
     - suites/ui/inventory/managing_inventory.yml
+    - suites/ui/inventory/showing_inventory_on_edit_product_page.yml
```

Comment:
Change the filename of this suite, since you have change the feature filename to be consistent